docs/getting_started/install.md

```diff
@@ -8,18 +8,25 @@ description: Learn how to install Shorebird command line interface on your machi
 
 To install the Shorebird command line interface (CLI):
 
-```
+## Mac/Linux
+
+```bash
 curl --proto '=https' --tlsv1.2 https://raw.githubusercontent.com/shorebirdtech/install/main/install.sh -sSf | sh
 ```
 
+## Windows
+
+```powershell
+$script = "https://raw.githubusercontent.com/shorebirdtech/install/main/install.ps1"
+powershell -exec bypass -c "(New-Object Net.WebClient).Proxy.Credentials=[Net.CredentialCache]::DefaultNetworkCredentials;iwr('$script')|iex"
+```
+
 :::info
-Installing Shorebird CLI requires `git`. We currently support only Mac and
-Linux. Windows is [coming
-soon](https://github.com/shorebirdtech/shorebird/issues/37).
+Installing Shorebird CLI requires `git`.
 :::
 
-This installs `shorebird` into `~/.shorebird/bin` and adds it to your `PATH`. It
-also installs a copy of Flutter and Dart inside
+This installs `shorebird` into `~/.shorebird/bin` and (on Mac and Linux) adds
```

Comment:
I presume this is not possible from powershell on windows?  (Maybe thats why rust-lang.org has an msi?)

Reply:
It's possible but requires running as an administrator. I opted to make this manual because 1) that's what Flutter does and 2) without this, we don't need to worry about running with elevated permissions.